core/imageroot/usr/local/agent/pypkg/agent/__init__.py

```diff
@@ -174,6 +174,41 @@ def run_restic(rdb, repository, repo_path, podman_args, restic_args, **kwargs):
 
     return subprocess.run(podman_cmd, **kwargs)
 
+def get_state_volume_args():
+    """Return a list of --volume arguments for Podman run and similar. The argument values
+    are built from the module etc/state-include.conf file, if available, otherwise return
+    the list of already existing volumes.
+    """
+    volume_args = {}
+    install_dir = os.environ['AGENT_INSTALL_DIR']
+    state_dir = os.environ['AGENT_STATE_DIR']
+    module_is_rootfull = os.geteuid() == 0
+    module_id = os.environ['MODULE_ID']
+    volume_args['_statedir'] = f"--volume={state_dir}:/srv/state"
+    if os.path.isfile(install_dir + '/etc/state-include.conf'):
+        # Mount Podman volumes by finding their names in the include file:
+        with open(install_dir + '/etc/state-include.conf', 'r') as incfile:
+            for iline in incfile:
+                if iline.startswith('volumes/'):
+                    sstart = len('volumes/')
+                    send = iline.find('/', sstart)
+                    svolume = iline[sstart:send]
+                    if not svolume:
+                        continue
+                    elif module_is_rootfull:
+                        dvolume = module_id + '-' + svolume
+                    else:
+                        dvolume = svolume
+
+                    volume_args[dvolume] = f"--volume={dvolume}:/srv/volumes/{svolume}"
+    else:
+        # TODO: As alternative default strategy,
+        #       we could mount already existing volumes
+        raise Exception("Not implemented")
```

Comment:
Can we cleanup this code?

```suggestion
        return list()
```